src/HypothesisTests.jl

```diff
@@ -30,13 +30,15 @@ using Combinatorics: combinations, permutations
 using Rmath: pwilcox, psignrank
 using Printf: @printf
 
-import StatsAPI
+using StatsAPI: StatsAPI
 using StatsAPI: HypothesisTest, confint, pvalue
 
 export testname, pvalue, confint, dof, nobs
 
-check_same_length(x::AbstractVector, y::AbstractVector) = if length(x) != length(y)
```

Comment:
Can you revert these? If we want to apply style changes, better do it in a separate PR anyway.

Reply:
I did